java/sage/VideoFrame.java

```diff
@@ -5650,6 +5651,40 @@ private int getMatchingLangIndex(String[] langOptions, String targetLang)
 
   private boolean selectDefaultSubpicLanguage()
   {
+	String defaultAudioLang = uiMgr.get("default_audio_language", "English"); 
+    MediaLangInfo mli = (MediaLangInfo) mediaLangMap.get(defaultAudioLang);
+    SubpictureFormat [] subs = this.currFile.getFileFormat().getSubpictureFormats();
+
+	/*
+	 * Loop thru all subtitle tracks of the mediafile to see if there is a forced subtitle 
+	 * track that matches the configured default audio language.  If it does it will 
+	 * select the track for playback.
+    */	 
+    for(int i = 0; i < subs.length; i++)
```

Comment:
I'm thinking you should do the forced subtitle selection only if they user hasn't configured a default subtitle. Otherwise, if they always use subtitles because they're hard of hearing, then your new code will break that and they'll only see the forced subs rather than all the subs.  Also, in the default subtitle selection code, you should update that to prefer the non-forced subs for the matching language in order to improve how that code works now that we know which are which.